src/views/AddPriceSingle.vue

```diff
@@ -412,18 +412,28 @@ export default {
     showUserRecentProofs() {
       this.userRecentProofsDialog = true
     },
-    handleProofSelected(proofId) {
-      this.addPriceSingleForm.proof_id = proofId
+    handleProofSelected(proof) {
+      this.addPriceSingleForm.proof_id = proof.id
+      this.addPriceSingleForm.date = new Date(proof.created).toISOString().split('T')[0]
```

Comment:
I'd rather keep it empty and let the user fill it in, no ? Or some kind of warning ?

Reply:
When setting a recent proof or proof from URL parameter ID I definitely intended the date to be automatically set like it already does when uploading a new proof, but maybe not to the proof upload date like it does here. Preferably it should set the date to when the picture was taken instead of today's date like the current running version of the site does when selecting a proof right now. However, without adding a column for file creation time in the database, it probably have to use some potentially expensive sql queries to determine the date based on other prices uploaded to the proof already, if it even has any price entries at all. I've tried to download proof images I uploaded with exif data that has been compressed and process into mebp files, but I was not able to find any of the original exif data or dates. Only the modification date of when I uploaded the file to Open Prices. It could be that the file's exif date is stripped when serving the file through the URL (and still available internally), but it appears that maybe that data is lost in the compression process.

Reply:
The date in the json response makes even less sense. It's before I even took the image. I'm assuming it's using a different time zone and it's actually 17:00 reflecting the date of the downloaded image:
```json
{
  "id": 6644,
  "file_path": "0007/2rO9xlT7Nn.webp",
  "mimetype": "image/webp",
  "type": "PRICE_TAG",
  "price_count": 1,
  "owner": "odinh",
  "created": "2024-05-21T15:00:46.692372Z"
}
```
Here's a screenshot showing the metadata of both the downloaded image and the original:
![image](https://github.com/openfoodfacts/open-prices-frontend/assets/150337574/da339769-4de1-4b15-9b0c-608bbaf8d256)

Reply:
yeah I think the server is on a strange timezone or something.
maybe we could add a field "date_taken" ? though sometimes we might not have this info, if an image is uploaded without the related metadata

Reply:
Maybe changing `created` in the json response from a offset-naive to a offset-aware timestamp would be a good idea to do instead, so it includes the timezone offset? And regarding the file metadata, could maybe make it default to the file's creation date instead, if `date_taken` is not available? Most files usually have a creation date even if they don't have the `date_taken` EXIF value. Another alternative would be to extract it from the filename like you could for the file in my example:
```
20240521_162617.jpg
```

Reply:
linked issues in the backend : 
- https://github.com/openfoodfacts/open-prices/issues/314
- https://github.com/openfoodfacts/open-prices/issues/107